server/config/subject_page.proto

```diff
@@ -103,20 +103,33 @@ message TopEventTileSpec {
 
 message Tile {
   // When adding a new TileType, make sure to update verify_tile in subject_page_configs_test.py
-  // with any required fields for your new TileType.
+  // with any required fields for your new TileType. Also, specify what block types can render
+  // the tiles.
   enum TileType {
     TYPE_NONE = 0;
+    // allowed in BlockTypes: TYPE_NONE
     LINE = 1;
+    // allowed in BlockTypes: TYPE_NONE
     BAR = 2;
+    // allowed in BlockTypes: TYPE_NONE
     MAP = 3;
+    // allowed in BlockTypes: TYPE_NONE
     SCATTER = 4;
+    // allowed in BlockTypes: TYPE_NONE
     BIVARIATE = 5;
+    // allowed in BlockTypes: TYPE_NONE
     RANKING = 6;
+    // allowed in BlockTypes: TYPE_NONE
     HIGHLIGHT = 7;
+    // allowed in BlockTypes: TYPE_NONE
     DESCRIPTION = 8;
+    // allowed in BlockTypes: DISASTER_EVENT
```

Comment:
let's group the ones for DISASTER_EVENT together at the bottom? they needn't be arranged by value.

Reply:
done